src/Courses/TopicGrades/PrintEverything.tsx

```diff
@@ -73,17 +72,24 @@ export const PrintEverything: React.FC<PrintEverythingProps> = () => {
                             readonly={true}
                         />
                         <h5>Problem {problem.problemNumber} Attachments</h5>
-                        {attachments.map((attachment) => {
-                            const cloudFilename = attachment.cloudFilename;
+                        {attachments && attachments.map((attachment) => {
+                            const { cloudFilename, userLocalFilename } = attachment;
+                            const cloudUrl = url.resolve(baseUrl.toString(), cloudFilename);
+                            if (!cloudFilename) {
```

Comment:
Shouldn't this be above the url resolve?
Also is this a viable thing to happen, maybe add a comment if this can happen otherwise log a warning or error